controller/cmd/ca-distributor/main.go

```diff
@@ -52,7 +51,8 @@ func main() {
 	stopCh := make(chan struct{})
 	ready := make(chan struct{})
 
-	go k8sAPI.Sync(ready)
+	// synchronously sync caches before starting the controller
```

Comment:
We should document *why* we must do it synchronously since I think the original idea was that controllers would do this asynchronously.